AmazonChimeSDK/AmazonChimeSDKTests/internal/audio/DefaultAudioClientObserverTests.swift

```diff
@@ -315,6 +315,16 @@ class DefaultAudioClientObserverTests: XCTestCase {
         wait(for: [expect], timeout: defaultTimeout)
     }
 
+    func testAttendeesPresenceChanged_attendeesDidJoinWithoutAudio() {
```

Comment:
Should this be included in this PR?

Reply:
It should not. Removed this, as well as some other "join without audio" test cases.